.github/workflows/e2e.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     timeout-minutes: 25
     env:
       FULLNODE_PUBKEY_KEEP: false
-      CGO_LDFLAGS: "-L/usr/local/lib -lbls-dash -lrelic_s -lgmp"
+      CGO_LDFLAGS: "-L/usr/local/lib -ldashbls -lrelic_s -lgmp -lminialloc"
```

Comment:
The Makefile uses the paths to the files in `third_party/bls-signatures/build` folder because we are building the library from the project.
However, CI uses another approach, we build the library using workflow action and copy the result into system (/usr/local/lib). 
This is the reason why CGO flags should be provided from OS environment variables.
Otherwise, we will look for the compiled files and sources into `third_party/bls-signatures/build`